src/views/v2/month/calendar-body/day/multiday-events/multiday-event.php

```diff
@@ -20,107 +22,25 @@
-	}
-
-	// If it ends this week, let's add the end class (right border).
-	if ( $event->ends_this_week ) {
-		$classes[] = 'tribe-events-calendar-month__multiday-event--end';
-	}
-
-	if ( $event->dates->end->format( 'Y-m-d' ) < $today_date ) {
-		$classes[] = 'tribe-events-calendar-month__multiday-event--past';
-	}
-}
 ?>
 <div class="tribe-events-calendar-month__multiday-event-wrapper">
 	<article <?php tribe_classes( $classes ); ?> data-event-id="<?php echo esc_attr( $event->ID ); ?>">
-		<div class="tribe-events-calendar-month__multiday-event-hidden">
-			<time
-				datetime="<?php echo esc_attr( $event->dates->start->format( Dates::DBDATEFORMAT ) ); ?>"
-				class="tribe-common-a11y-visual-hide"
-			>
-				<?php echo esc_attr( $event->dates->start->format( Dates::DBDATEFORMAT ) ); ?>
-			</time>
-			<a
-				href="<?php echo esc_url( $event->permalink ); ?>"
-				class="tribe-events-calendar-month__multiday-event-hidden-link"
-				data-js="tribe-events-tooltip"
-				data-tooltip-content="#tribe-events-tooltip-content-<?php echo esc_attr( $event->ID ); ?>"
-				aria-describedby="tribe-events-tooltip-content-<?php echo esc_attr( $event->ID ); ?>"
-			>
-				<?php if ( $event->featured ) : ?>
-					<em
-						class="tribe-events-calendar-month__multiday-event-hidden-featured-icon tribe-common-svgicon tribe-common-svgicon--featured"
-						aria-label="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
-						title="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
-					></em>
-				<?php endif; ?>
-				<h3 class="tribe-events-calendar-month__multiday-event-hidden-title tribe-common-h8">
-					<?php
-					// phpcs:ignore
-					echo $event->title;
-					?>
-				</h3>
-			</a>
-		</div>
-		<?php if ( $should_display ) : ?>
-			<div class="tribe-events-calendar-month__multiday-event-bar">
-				<div class="tribe-events-calendar-month__multiday-event-bar-inner">
-					<?php if ( $event->featured ) : ?>
-						<em
-							class="tribe-events-calendar-month__multiday-event-bar-featured-icon tribe-common-svgicon tribe-common-svgicon--featured"
-							aria-label="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
-							title="<?php esc_attr_e( 'Featured', 'the-events-calendar' ); ?>"
-						></em>
-					<?php endif; ?>
-					<h3 class="tribe-events-calendar-month__multiday-event-bar-title tribe-common-h8">
-						<?php
-						// phpcs:ignore
-						echo $event->title;
-						?>
-					</h3>
-				</div>
-			</div>
-			<?php if ( $is_first_appearance ) : ?>
-				<?php $this->template( 'month/calendar-body/day/calendar-events/calendar-event/tooltip', [ 'event' => $event ] ); ?>
-			<?php endif; ?>
-		<?php endif; ?>
+		<?php $this->template( 'month/calendar-body/day/multiday-events/multiday-event/hidden', [ 'event' => $event ] ); ?>
+		<?php
+		$this->template(
+			'month/calendar-body/day/multiday-events/multiday-event/bar',
+			[
+				'event'               => $event,
```

Comment:
:warning: Array double arrow not aligned correctly; expected 12 space(s) between "'event'" and double arrow, but found 15.

<sub>hash: 159897ec1a759316ab616b028c8259d5</sub>